pom.xml

```diff
@@ -65,15 +65,14 @@
                 <dependency>
                   <groupId>org.junit.jupiter</groupId>
                   <artifactId>junit-jupiter-engine</artifactId>
-                  <version>5.2.0</version>
+                  <version>5.11.0</version>
```

Comment:
update surefire plugin version instead, and you can get rid of this additional dependencies.